scout/models/case/case_loading_models.py

```diff
@@ -436,6 +436,9 @@ class CaseLoader(BaseModel):
     smn_tsv: Optional[str] = None
     sv_rank_model_version: Optional[str] = None
     synopsis: Optional[Union[List[str], str]] = None
+    status: Optional[
+        Literal["prioritized", "inactive", "ignored", "active", "solved", "archived"]
```

Comment:
Would be nice to reuse the constant here, right? I guess https://github.com/Clinical-Genomics/scout/blob/a7b8afbe605f9bb10a3e77c316ef349e195f3789/scout/constants/case_tags.py#L68